packages/css/test/index.ts

```diff
@@ -206,13 +217,44 @@ test('works with functional arguments', () => {
 
 test('supports functional values', () => {
   const result = css({
-    color: t => t.colors.primary,
+    color: (t) => t.colors.primary,
   })(theme)
   expect(result).toEqual({
     color: 'tomato',
   })
 })
 
+test('returns default key when accessing object value with default', () => {
+  const result = css({
+    color: 'colorScale',
+  })(theme)
+  expect(result).toEqual({
+    color: 'color500',
+  })
+})
+
+test('returns nested key when accessing key from object value with default', () => {
+  const result = css({
+    color: 'colorScale.100',
+  })(theme)
+  expect(result).toEqual({
+    color: 'color100',
+  })
+})
+
```

Comment:
It'd be nice to have an additional test showing this working with a variant and returning an object